gravitee-am-ui/src/app/domain/settings/password-policy/pass-policy-status/password-policy-status.component.ts

```diff
@@ -74,8 +74,10 @@ export class PasswordPolicyStatusComponent implements OnChanges, OnDestroy {
       this.rules = this.getRules(changes['policy'].currentValue);
       // debounce if we're checking with the backend, but not if we're doing everything in-browser
       if (this.rules.some((x) => x.id === 'backend-check')) {
+        console.log('piping backend check');
         this._passwordInput.pipe(debounceTime(250), takeUntil(this._backendCheckDisabled)).subscribe((pass) => this.checkRules(pass));
       } else {
+        console.log('piping no backend check');
```

Comment:
question: do we really need this 2 logs ? If so, maybe it should be log using DEBUG level and message could be "evaluate password calling the backend" or "evaluate password locally" :thinking: